packages/valory/skills/abstract_round_abci/abci_app_chain.py

```diff
@@ -190,11 +190,11 @@ def get_paths(
                     accumulated_post_conditions,
                 )
                 if len(diff) != 0:
-                    raise ValueError(  # pragma: no cover
+                    raise ValueError(
                         f"Pre conditions '{diff}' of app '{next_app}' not a post condition of app '{current_app}' or any preceding app in path {path}."
                     )
             else:
-                raise ValueError(  # pragma: no cover
+                raise ValueError(
```

Comment:
I left this `#pragma: no cover` because I don't think this can be reached. 
https://github.com/valory-xyz/open-autonomy/blob/48f606e7fff733531a4c5463256af0bfe2b3c1cb/packages/valory/skills/abstract_round_abci/abci_app_chain.py#L176-L181
It requires that the `next_final_state = None` in one of the paths prior to the last, such inconsistency however are dealth with via the `_MetaAbciApp` or caught earlier here
https://github.com/valory-xyz/open-autonomy/blob/48f606e7fff733531a4c5463256af0bfe2b3c1cb/packages/valory/skills/abstract_round_abci/abci_app_chain.py#L102-L117